ixdiagnose/config.py

```diff
@@ -13,6 +13,7 @@ class Configuration:
             'debug_path': {'type': ['string', 'null']},
             'exclude_artifacts': {'type': 'array', 'items': {'type': 'string'}},
             'exclude_plugins': {'type': 'array', 'items': {'type': 'string'}},
+            'extra': {'type': 'object'},
```

Comment:
We should probably specify something like this as well so it validates that the values provided in this dict are actually string based and not integer/list or something else
```
    'type': 'object',
    'patternProperties': {
        '.*': {
            'type': 'array',
            'items': {'type': 'string'},
        }
    },
```